packages/cli/src/commands/dao_cmds/new.js

```diff
@@ -83,37 +79,26 @@ export const task = async ({
           })
         },
       },
-      {
-        title: `Fetching template ${bold(template)}@${templateVersion}`,
-        task: async () => {
-          repo = await getApmRepo(web3, template, apmOptions, templateVersion)
-        },
-        enabled: () => !templateInstance,
-      },
       {
         title: 'Create new DAO from template',
         task: async ctx => {
-          daoAddress = await newDao({
```

Comment:
The interfaces on all requests to the toolkit functions change from requesting options like: `web3`, `apmOptions`, `repo`, `wsProvider`, `gasPrice`. To only ask for an `environmnet` option that will be setup by the toolkit.